common-java-libs/gradle.properties

```diff
@@ -1,2 +1,19 @@
+#
+# Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
 group=org.wso2.apk
-version=0.0.1-SNAPSHOT
+version=0.1.0-SNAPSHOT
+openapiGeneratorVersion=6.2.1
```

Comment:
Shall we move this property to `java.gradle`?

Reply:
Fixed with [c8fd2b8](https://github.com/wso2/apk/pull/110/commits/c8fd2b87c31dcae773015688b84f53279bee65a0)